test/test.sh

```diff
@@ -9,8 +9,21 @@ log() {
   echo >&2 "[$( date +%Y-%m-%dT%H:%M:%S] ) $*"
 }
 
+start_cluster_run_tests() {
+  root_dir=$1
+  $root_dir/yb-ctl $flag_data_dir start
+  $root_dir/yb-ctl $flag_data_dir add_node
+  $root_dir/yb-ctl $flag_data_dir stop_node 1
+  $root_dir/yb-ctl $flag_data_dir start_node 1
+  $root_dir/yb-ctl $flag_data_dir stop
+}
+
 yb_data_dirs=(
-  "$HOME/yugabyte-data"
+  "/tmp/yugabyte-data"
+)
```

Comment:
Not sure why we need an array here if we're only using one data dir anyway.

Reply:
I assumed your intention when you set this up was that we might add multiple data dirs